platform/src/common/components/AQNumberCard/index.jsx

```
@@ -188,7 +188,9 @@ const AQNumberCard = () => {
                     </div>
                   </div>
                   <div className="text-gray-700 text-[28px] font-extrabold">
-                    {reading !== null ? reading.toFixed(2) : '--'}
+                    {typeof reading === 'number' && !isNaN(reading)
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)